CONTRIBUTING.md

```diff
@@ -1,79 +1,114 @@
 
-We welcome Pull Requests (PR's) as well as having people working on fixing it's current issues. If you are coming to this project new.
-If you notice an issue with this app, please feel free to open an issue.
+This project welcomes contributors to help improve its codebase by identifying shortcomings in the project, highlighting bugs, and issues in the documentation.
 
-A lot of developers prefer asking if they could contribute or be assigned a task as a reply to a reported issue in the Issues tab. This is great.
+## What to contribute
 
-Typically, if you think it will take you longer than 72 hours after having been assigned to the issue you will work on to submit a PR, please
-let us know that way issues do not go stale.
+Anyone with basic programming skills down to grammer skills can contribute immensely to this project.
+Areas requiring contribution includes:
 
-Some tasks may not be beginner friendly even if tagged `good first issue`.. so try and judge the task accordingly.
-If you are a beginner, there are smaller tasks a beginner can work on such as style issues.
+- **Error checks**: Identify any mispellings or errors in documentation? 
 
-You should already be familiar with React 17 and or 18, JavaScript ES5/ES6/ES7.
+- **Documentation**: Possess strong documentation skills, and believe you can help elevate this projects' documentation even better? 
 
-## Style
+- **Code contribution**: Identify a bug in this app or thought of a new feature that could plunge this app to a new dimension? 
+
+## How to contribute
+
+Developers often derive more convenience in seeking permission from the project to contribute, so these next headings will get you started in no time contributing.
+
+## Select an issue
+
+If you are not sure what to work on, review the issues list first. There are also TODO's listed in the included [TODO.md](./TODO.md) file. 
+
+@gbowne1 and other project maintainers will assign users to issues on a first come, first serve basis. If you would like to work on an issue, feel free to indicate by tagging the mentioned persons.
+
+## Create an issue
+
+If you notice a typo, slight bug in the code and there are no lodged complaints on this yet, select the issues icon at the top, and on the right side of the next page select new issues and lodge your complain.
 
-I have not created an official style guide yet. I also have not created a Figma or layout. If anyone would like to create a style guide for us. Create a Discussion. I have typically let the community help
+## Fork this repository
 
-## Development
+After choosing what to work on, move to fork and setup your own repository.
 
-If you are not sure what to work on, review the issues list. There are also TODO's that will listed in the included [TODO.md](./TODO.md) file.
+On the project's repository page, look to the right side of the page to locate a fork tab, click on this icon, and this would redirect you to a new page with further instructions. 
```

Comment:
This might be a great place in the future to provide instructions for both cloning and forking.  They are similar process, but work differently especially when it comes to version control.  "Fork" was meant to be used for a derivitive project, and also meant for developers who wished to make their own version of a project, or a developer could use parts of a project to create their own project within the terms of the original project's license of course..   "Clone" is just that, a `clone` of the project, not meant to be used for creating any derivitive project that would be different from the main repository.

Reply:
I felt this would make this page more choked up, i wanted to provide information to this on the devsetup page, or what do you think?

Reply:
Yes, @Chizobaonorh, that information for would actually be better for the DEVSETUP.md file